dojo/fixtures/test_type.json

```diff
@@ -648,11 +648,11 @@
             "name": "Risk Recon API Importer"
         },
         "model": "dojo.test_type",
-		"pk": 184
+		     "pk": 184
     },
     {
         "fields": {
-            "name": "DrHeader JSON Importer"
+            "name": "Semgrep JSON Report"
```

Comment:
looks like you're overwriting drheader importer?

Reply:
Yes, was a bad commit. 
We do not need to override it, but add a new PK assign to Semgrep.